Sources/ITwinMobile/ITMApplication.swift

```diff
@@ -270,9 +290,8 @@ open class ITMApplication: NSObject, WKUIDelegate, WKNavigationDelegate {
     }
 
     /// Loads the app config JSON from the main bundle.
-    /// Override this function in a subclass in order to add custom behavior.
     /// - Returns: The parsed contents of ITMAppConfig.json in the main bundle in the directory returned by `getWebAppDir`.
-    open func loadITMAppConfig() -> JSON? {
+    func loadITMAppConfig() -> JSON? {
```

Comment:
I think we should still allow this to be overridden. I'm pretty sure SYNCHRO uses a different name for the json file. Don't forget to put the comment back also.

Reply:
Or is that not allowed due to it being called from init?

Reply:
If that's the case, then this probably needs to be an overridable class function.

Reply:
Restored to previous definition and comment.